projects/tuist/fixtures/macos_app_with_extensions/Project.swift

```diff
@@ -0,0 +1,44 @@
+import ProjectDescription
+
+/**
+These are necessary for the compiler to find the Workflow's SDK and link against it.
+**/
+let workflowExtensionSettings: SettingsDictionary = [
```

Comment:
should we try to set these by default if we know it's a mac app with extensions? At least for `.recommended` settings? (I'd say out of scope for this PR but if it makes sense, might be worth creating an issue)

Reply:
I don't think so because some of the settings in this group are specific to extensions of type Workflow. I'd leave it as it is, and if we notice there are some of them that become common across our userbase, then I'd consider auto-generating them.

Reply:
Yeah, I think that most extension types have some customisations specific to them. Might be worth adding some support (or at least documentation, as here) for specific extension types as they become apparent.